nss-day-cohort-41/tabloid-fullstack-first-amendment-dancers-andy
# Tag to post pm

# Description
Tags may be added or deleted from the Post Details view. If a post has any associated tags, they will be displayed in the upper right corner of the post details card

## Type of change

# Testing Instructions
As a logged in user, navigate to the details page of a post 'owned' by that user. Tag management is only available to a post's author. The Tag Management button is at the bottom of the details card, next to comments and edit. Clicking Tag Management will open the list of available tag, displayed as checkboxes. If there are any tags associated with the post, those tags will appear checked. The Save button is disabled until the first change is made. Click a checkbox (only once, YAY!) to change its state. If an initially checked tag is deselected, it will be deleted when the save button is clicked. Any checked tags will be saved to the post (If they were initially checked, they are ignored unless their state has changed). The API will not write duplicate tags.
# Checklist:
